packages/jss-plugin-props-sort/src/index.js

```diff
@@ -11,8 +14,8 @@ export default function jssPropsSort() {
 
     const newStyle = {}
     const props = Object.keys(style).sort(sort)
-    for (const prop in props) {
-      newStyle[props[prop]] = style[props[prop]]
+    for (const prop of props) {
```

Comment:
this generates a lot of code, which is an overkill for a library: https://babeljs.io/repl/#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=GYewTgBAFAxiB2BnALhADmEaImOzaiAlBAN4BQEV8ApgO4DKyAngDY0DaGWAuhALwQUbTtzQ9yAXyA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&sourceType=module&lineWrap=false&presets=es2015%2Ces2016%2Ces2017%2Cstage-0&prettier=false&targets=&version=6.26.0&envVersion=

Reply:
Didn't know it generated that much code. I replaced it with a forEach now. The reason I changed it from a `for ... in` is because it's not supposed to be used with arrays.